### ISSUE TITLE
Install OpenShift Pipelines 0.5 Operator and Tekton CRDs

### ISSUE THREAD

**Issue by danielhelfand** [openshift-labs/lab-tekton-pipelines#6]
The workshop currently uses [custom resource definitions](https://github.com/openshift-labs/lab-openshift-pipelines-with-tekton/tree/2ced1fbbfadfb3ed4ccd793b2b8a171984ceacef/.workshop/resources/operator) to install the OpenShift Pipelines Operator 0.5. The resource definitions here do not appropriately install the operator, and this doesn't allow the user to proceed with the workshop.

To solve this, the workshop needs corrected resource definitions to properly install the operator, or the operator needs to be installed globally on the cluster so that users have the ability to create the custom resource definitions for Tekton.

**Comment by danielhelfand** [comment]
This is currently being handled by manually installing the operator on the cluster and is detailed in the `README`.

**Comment by GrahamDumpleton** [comment]
Operator installation seems to be totally broken when done on fresh cluster if so happen to have deployed the spawner yourself first before looking at enabling the operator. When do go to operator hub it shows as already installed but CRDs do not exist. When try and uninstall it, you can't as gives error:

```
catalogsourceconfigs.operators.coreos.com "installed-custom-openshift-operators.openshift-marketplace" not found
```

So suspect with the history of experimental changes you had in cluster you had the operator working, but then never tested in fresh cluster to see whether steps worked.

Had to run `.workshop/scripts/delete-resources.sh` to delete the global resources you created. Could then install the operator properly from operator hub in the web console as cluster admin.

Suspecting the problem is that remaining files in `.workshop/resources` should exist and directory should be removed. Should rely on it being installed from web console only. Am removing the `.workshop/resources` directory for now to test and see what happens.

**Comment by GrahamDumpleton** [comment]
Removing all of `.workshop/resources` then results in the error about `pipelines-privileged-user` being missing because that was created as cluster role. I wouldn't have created that as a global resource from `.workshop/resources` but would have created it from extra resource config map and give it a name which includes the project namespace. The role binding would then use that. Homeroom will clean it up using Kubernetes GC when session is deleted, even though is a global resource.

**Comment by danielhelfand** [comment]
This was solved by [92de461](https://github.com/openshift-labs/lab-tekton-pipelines/commit/92de461d0b3f22fd294f4ce5a26a9641768a0d8e)
